utils/controller.js

```diff
@@ -0,0 +1,26 @@
+const { models } = require('../dal/database');
+const { StatusCode } = require('../shared/constants');
+
+/**
+ * Helper method to insert a Product into the collection
+ */
+exports.insertProduct = function(body, barcode, res) {
+  let newProduct = new models.Product ({
+    barcode: barcode || body.barcode,
+    name: body.product_name,
+    category: body.category.split('>').map(Function.prototype.call, String.prototype.trim),
```

Comment:
Can we change the category structure as per our discussion for which we referenced this:

https://docs.google.com/document/d/1NLGdPTkV9savb8d1MPY2bYVERdfGwiGpd_PqaP-m3oI/edit